### ISSUE TITLE
How to use sync feature?

### ISSUE THREAD

**Issue by fbartels** [huytd/pomoday-v2#42]
Hi,

I saw that there is now a new "login" command, after which tasks should be synced to a central server. There is no mention of what values to enter into this view and also no register command.

How can one use it? How to use it on the self hosted version?

**Comment by huytd** [comment]
Hi, the sync feature has been implemented on the frontend but the backend isn't finished yet. I'll have the instruction for it later when everything is ready.

Thanks!

**Comment by thangngoc89** [comment]
@huytd I'm curious if you looked into automerge or other CRDT syncing projects for this feture? This app is a perfect project for using those

**Comment by huytd** [comment]
@thangngoc89 Yes I did looked at `automerge` and PouchDB at first, but for some reason, the BE dev in my team decided to go with the self-developed solution first :D we might added it (and other) later as an extension or something.

**Comment by thangngoc89** [comment]
I have some experience with synching using both and I’d say a custom built solution is the worst option :) . At least basing it on CRDT or OT

**Comment by huytd** [comment]
Yeah I agree with you about that :D there's some other reason behind it, but we mostly want to try out to see how far we could go at first :D

**Comment by deepDiverPaul** [comment]
are there any news on that? some sort of sync across devices would be perfection.

wonderful tool!!

**Comment by Tim-Dixon** [comment]
+1 For a sync feature! Love the app, but definitely need to be able to access the list from multiple devices! Looking forward to it!

**Comment by huytd** [comment]
Thank guys! I just published a sync API specs, if you want to build the server yourself https://github.com/huytd/pomoday-v2/wiki/Pomoday-Backend-API-Specification

The current frontend is already implemented to work if you follow this backend specs.

**Comment by tuananh** [comment]
@huytd is there a place (wiki) to list all the opensource backend implementation? I'm interested in building one with Cloudflare workers.

Very cool project btw. Thanks.

**Comment by huytd** [comment]
@tuananh hey, thanks. About your question, there's no actual list of backend implementations right now, a few people tried to implement the backend themselves before I published the backend specs, but they never released it.
